VOTable.tex

```diff
@@ -863,16 +866,16 @@ \subsection{\elem{RESOURCE} Element}
 should exist in any of its sub-elements. A \elem{RESOURCE} without
 this attribute {\em may} however have no \elem{DATA} sub-element.
 
-\elem{RESOURCE} elements qualified by
-\attrval{type}{meta} are also used to transmit
+For example, a \elem{RESOURCE} qualified by
+\attrval{type}{meta} can be used to transmit
 MIVOT (Model Instances in VOTables) annotations \citep{2023ivoa.spec.0620M}.
 MIVOT defines a syntax to map VOTable
 contents to any model serizalized in VO-DML \citep{2018ivoa.spec.0910L}.
 It operates as a bridge between models and data that associates VOTable
 metadata to data model entities, possibly adding advanced metadata not
 representable in plain VOTable.
-MIVOT annotations have their own XMLnamespace;
-they have no impact on the VOTable schema.
+MIVOT annotations have their own XMLnamespace.  The VOTable schema allows MIVOT,
+and any valid XML, in a \elem{RESOURCE}.
```

Comment:
This should say something like  _"and any elements from a foreign namespace"_  not _"and any valid XML"_ here.  Non-VOTable elements in the VOTable namespace are not allowed.